commands/utils/params.go

```diff
@@ -476,7 +477,11 @@ func extractClientServerParams() (*coreconfig.ServerDetails, *ClientInfo, error)
 	if err != nil {
 		return nil, nil, err
 	}
-	return &server, clientInfo, nil
+	git := Git{
+		ClientInfo:     *clientInfo,
+		AggregateFixes: AggregateFixesDefaultValue,
```

Comment:
Default is false, initialize is unnecessary here, so you can return `*ClientInfo`.